doc/admin-api.md

```diff
@@ -716,6 +717,33 @@ Content-Type: text/plain
 
 [Back to TOC](#Table-of-Contents)
 
+## Plugin
+
+*API*：/apisix/admin/plugins/{plugin_name}
+
+*Description*: plugin
+
+> Request Methods:
+
+|Method      |Request URI|Request Body|Description        |
+|---------|-------------------------|--|------|
+|GET      |/apisix/admin/plugins/list|NULL|Fetch resource list|
```

Comment:
@nic-chen @starsz Does manager-api supports this feature?

Reply:
sure
`/apisix/admin/plugins` returns plugin name list like this.

Reply:
ok